lib/api/errors/error-handler.ts

```
@@ -0,0 +1,162 @@
+  }
+
+  // Handle Prisma errors by checking for properties that Prisma errors typically have
+  if (
+    typeof error === "object" &&
+    error !== null &&
+    "code" in error &&
+    typeof error.code === "string" &&
+    error.code.startsWith("P")
+  ) {
+    // This is likely a Prisma error
+    const prismaError = error as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)